### ISSUE TITLE
Current extraction logic produces duplicates

### ISSUE THREAD

**Issue by regulartim** [intelowlproject/GreedyBear#404]
Currently the jobs that extract the data from Elastic run every 10 minutes. By default, the query asks for all data from the last 11 minutes. This produces an overlap of 1 minute. So each entry from Elastic that has a timestamp between `h:59:00` and `h+
1:00:00` will potentially be extracted twice, e.g. both the celery job starting at 8:00:00 and the one starting at 8:10:00 will extract entries from 7:59:30.  

One possible solutions would be to asks for the data from the last 10 minutes. But depending on how accurately celery times the job execution, some data might not be extracted at all. 

Another idea is to use kind of a sliding window that always extracts the entries from the last completed 10 minute period, e.g. the celery job starting at 8:10:00 will extract every entry with a timestamp t for which 8:00:00 <= t < 8:10:00 applies.

**Comment by mlodic** [comment]
yep, last option would have probably be the best way to solve it

**Comment by regulartim** [comment]
Fixed.
